app/assess/templates/macros/fund_dashboard_summary.html

```diff
@@ -53,6 +53,13 @@ <h3 class="govuk-body govuk_summary-card__title govuk-!-font-size-24">{{round_na
             View all {% if is_active_status %}active{% else %}
             closed{% endif %} assessments
         </a>
+        {% if round_application_fields_download_available and access_controller.is_lead_assessor%}
+        </br>
+        </br>
+        <a class="govuk-link" data-qa="dashboard_export" href="{{ export_href }}">
+            Export Applicant Information
```

Comment:
Did this come from content? Seems a tiny bit inconsistent with:

```View all active assessments```   (sentence case)

vs

```Export Applicant Information```  (title case)